packages/gatsby-plugin-feed/src/createFeed.js

```diff
@@ -0,0 +1,64 @@
+import url from 'url'
+import { defaultsDeep, get } from 'lodash'
+import RSS from 'rss'
+
+const defaultOptions = {
+  generator: `Gatsby`,
+  title: `RSS Feed`,
+}
+
+export default function createFeed({
+  query: {
```

Comment:
I have questioned if there's a reason why query data shouldn't just be merged in with other parameters. Does it need to live under the key `query` when most of the data is being derived from that object? I don't care either way, It's just another potential simplification I thought of.